actions/upload_sessions.go

```diff
@@ -155,7 +155,13 @@ func (usr *UploadSessionResource) Create(c buffalo.Context) error {
 		fmt.Println(err)
 	}
 
-	privateKeys := services.EthWrapper.GenerateKeys(len(mergedIndexes))
+	privateKeys, err := services.EthWrapper.GenerateKeys(len(mergedIndexes))
+	if err != nil {
+		err := errors.New("Could not generate eth keys: " + err.Error())
+		fmt.Println(err)
```

Comment:
For this error and the other new error added to this file, I don't send the error to sentry because it gets captured in the generateKeys method.